hardware/electrical/power_supply.kicad_sch

```diff
@@ -5,6 +5,2413 @@
+    (property "Value" "MCP1700-3302E_SOT23" (id 1) (at 247.65 33.02 0))
+    (property "Footprint" "Package_TO_SOT_SMD:SOT-23" (id 2) (at 247.65 32.385 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "http://ww1.microchip.com/downloads/en/DeviceDoc/20001826D.pdf" (id 3) (at 247.65 38.1 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid eddd847b-f6a7-48e9-b41e-11ee26ac05f2))
+    (pin "2" (uuid c9e57b5b-fde8-4b71-b54f-de1ab72cdbde))
+    (pin "3" (uuid 4bacdb5a-b8e7-4e97-be5e-77857f9232e7))
+  )
+
+  (symbol (lib_id "power:PWR_FLAG") (at 124.46 45.72 0) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid b7605f31-ed62-4ed8-a1de-2dc59146f458)
+    (property "Reference" "#FLG03" (id 0) (at 124.46 43.815 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Value" "PWR_FLAG" (id 1) (at 119.38 41.91 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Footprint" "" (id 2) (at 124.46 45.72 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 124.46 45.72 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid 9867ad9d-3c91-44b2-b8b6-532a4c9785a4))
+  )
+
+  (symbol (lib_id "Device:R") (at 73.66 46.99 0) (mirror y) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid bf5e5076-35f6-4fda-a2d2-5da2e0063af0)
+    (property "Reference" "R2" (id 0) (at 76.2 46.99 90))
+    (property "Value" "470" (id 1) (at 71.12 46.99 90))
+    (property "Footprint" "Resistor_SMD:R_0805_2012Metric_Pad1.20x1.40mm_HandSolder" (id 2) (at 75.438 46.99 90)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 73.66 46.99 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid 2dab0936-fc52-40d8-b2d8-fb4328eaea23))
+    (pin "2" (uuid b018f572-4a86-41a5-ac4c-438f48837ce0))
+  )
+
+  (symbol (lib_id "Device:R") (at 134.62 45.72 90) (mirror x) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid c400ed91-b57e-453d-aa06-47831371a51b)
+    (property "Reference" "R4" (id 0) (at 134.62 43.18 90))
+    (property "Value" "100" (id 1) (at 134.62 48.26 90))
+    (property "Footprint" "Resistor_SMD:R_0805_2012Metric_Pad1.20x1.40mm_HandSolder" (id 2) (at 134.62 43.942 90)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 134.62 45.72 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid c4bcc549-7b88-43ce-bf7e-9ec909bc20e9))
+    (pin "2" (uuid 73fce98c-6375-4aa7-a3a6-b87c94600452))
+  )
+
+  (symbol (lib_id "Device:D_Schottky") (at 180.34 43.18 90) (unit 1)
+    (in_bom yes) (on_board yes) (fields_autoplaced)
+    (uuid ce4bf817-a5c1-41e0-ab7b-a8279ac2c19d)
+    (property "Reference" "D2" (id 0) (at 182.88 42.2274 90)
+      (effects (font (size 1.27 1.27)) (justify right))
+    )
+    (property "Value" "D_Schottky" (id 1) (at 182.88 44.7674 90)
```

Comment:
What voltage Schottky diode should this be?

Reply:
No higher than 1.7V, so any schottky diode will do.